.circleci/config.yml

```diff
@@ -0,0 +1,36 @@
+jobs:
+  build:
+    executor: python/default
+    steps:
+      - checkout
+      - python/load-cache
+      - python/install-deps
+      - python/save-cache
+      - run:
+          command: |
+            # pytest would be a dep in requirements.txt
+            python -m pytest --cov ophys_etl_pipelines --cov-report xml
+            bash <(curl -s https://codecov.io/bash) -t ${CODECOV_TOKEN}
+          name: Test
+  lint:
+    executor: python/default
+    steps:
+      - checkout
+      - run: 
+          command: |
+            pip install flake8
```

Comment:
Dan found a flake8 orb (see croissant repo) that has more configuration and will be less buggy I think

Reply:
Since I'm not familiar with CircleCI orbs, and am not sure how to configure the orb that Dan had [mentioned ](https://circleci.com/orbs/registry/orb/plushcare/flake8) I'm probably going to punt on this issue.